compiler/x/codegen/OMRInstOpCode.hpp

```diff
@@ -531,6 +536,8 @@ class InstOpCode: public OMR::InstOpCode
                   supported = target->supportsFeature(OMR_FEATURE_X86_AVX512BW);
                if (supported && flags & X86FeatureProp_EVEX256RequiresAVX512DQ)
                   supported = target->supportsFeature(OMR_FEATURE_X86_AVX512DQ);
+               if (supported && flags & X86FeatureProp_EVEX128RequiresAVX512CD)
```

Comment:
Should this be `X86FeatureProp_EVEX256RequiresAVX512CD` ?

Reply:
yes